extensions/arc/src/ui/dashboards/postgres/postgresExtensionsPage.ts

```diff
@@ -54,30 +58,38 @@ export class PostgresExtensionsPage extends DashboardPage {
 			CSSStyles: { ...cssStyles.text, 'margin-block-start': '0px', 'margin-block-end': '0px' }
 		}).component());
 
-		const info = this.modelView.modelBuilder.text().withProperties<azdata.TextComponentProperties>({
+		const info = this.modelView.modelBuilder.text().withProps({
 			value: loc.extensionsFunction,
 			CSSStyles: { ...cssStyles.text, 'margin-block-start': '0px', 'margin-block-end': '0px' }
 		}).component();
 
-		const link = this.modelView.modelBuilder.hyperlink().withProperties<azdata.HyperlinkComponentProperties>({
-			label: loc.extensionsLearnMore,
-			url: 'https://docs.microsoft.com/azure/azure-arc/data/using-extensions-in-postgresql-hyperscale-server-group',
+		const link = this.modelView.modelBuilder.hyperlink().withProps({
+			label: loc.learnMore,
+			url: 'https://www.postgresql.org/docs/12/external-extensions.html',
```

Comment:
Would be nice to link to the version of the documentation that matches the engine version, rather than always using 12.